### DISCUSSION TITLE
Public vaultwarden

### DISCUSSION THREAD

**Discussion by R0GGER** [dani-garcia/vaultwarden#4142] [General]
I came across a public vaultwarden via DuckDuckGo.    
Are you related? If not, are any donations transferred to @dani-garcia? And maybe people should be warned you're not relate?    

*I don't want to advert this website/service.

**Comment by BlackDex** [comment]
No Vaultwarden does not and will not ever host any public available instance.

Any publicly available instance has no link to this project in any way except that it uses Vaultwarden of course.

**Comment by Y0ngg4n** [comment]
@R0GGER Hosting a public vaultwarden.
We clearly state on our website: 
`This is not an official vaultwarden instance, maintained by the repository owners.` 
We also changed the Logo so there is really no way to missunderstand this. We are not related to the vaultwarden repository maintainers except that we donate to them monthly. If you would have read the website there is multiple times mentioned that we are not maintained by the repository owners. So i dont see any reason why people should get "warned". 
You can even see that the Donations are transfered to me and not to @dani-garcia.
